output.js

```diff
@@ -40,7 +40,7 @@ function status(config, state) {
     let status_str;
     for (let running_show = running.length;running_show >= 0;running_show--) {
         const running_str = (
-            running.slice(0, running_show).map(({tc}) => tc.name).join(' ')
+            running.slice(0, running_show).map(task => task.name).join(' ')
```

Comment:
I'm not sure I fully understand this line. Looking at the other changes in the PR it seems like `task.tc.name` holds the proper name like here: https://github.com/boxine/pentf/pull/97/files#diff-2375abc0904a66fd5a4a879c9b129fadR25

Did we change the shape of the `task` object?

Reply:
No, the task object has stayed the same. However, here we used to output the test case (`tc`) name. Now, we output the task name. By default, test case and task name are the same.
But with `--repeat` (or future retesting), the test name is unchanged (something like `selftest_browser`), but the task name will include the repetition, e.g. `selftest_browser[4]`